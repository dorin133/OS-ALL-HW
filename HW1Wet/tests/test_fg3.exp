smash> smash> smash> smash> smash> smash> sleep 300& : 111005
smash: got ctrl-C
smash: process 111005 was killed
smash> [1] sleep 100& : 111003 2 secs
[2] sleep 200& : 111004 2 secs
[4] sleep 400& : 111006 2 secs
[5] sleep 500& : 111007 2 secs
smash> sleep 100& : 111003
smash: got ctrl-C
smash: process 111003 was killed
smash> [2] sleep 200& : 111004 4 secs
[4] sleep 400& : 111006 4 secs
[5] sleep 500& : 111007 4 secs
smash> sleep 200& : 111004
smash: got ctrl-C
smash: process 111004 was killed
smash> [4] sleep 400& : 111006 6 secs
[5] sleep 500& : 111007 6 secs
smash> sleep 500& : 111007
smash: got ctrl-C
smash: process 111007 was killed
smash> [4] sleep 400& : 111006 8 secs
smash> sleep 400& : 111006
smash: got ctrl-C
smash: process 111006 was killed
smash> smash> smash: sending SIGKILL signal to 0 jobs:
