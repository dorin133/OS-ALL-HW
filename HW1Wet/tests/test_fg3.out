smash> smash> smash> smash> smash> smash> sleep 300& : 111009 
smash: got ctrl-C
smash: process 111009 was killed
smash> [1] sleep 100& : 111003 2 secs
[2] sleep 200& : 111006 2 secs
[4] sleep 400& : 111010 2 secs
[5] sleep 500& : 111011 2 secs
smash> sleep 100& : 111003 
smash: got ctrl-C
smash: process 111003 was killed
smash> [2] sleep 200& : 111006 4 secs
[4] sleep 400& : 111010 4 secs
[5] sleep 500& : 111011 4 secs
smash> sleep 200& : 111006 
smash: got ctrl-C
smash: process 111006 was killed
smash> [4] sleep 400& : 111010 6 secs
[5] sleep 500& : 111011 6 secs
smash> sleep 500& : 111011 
smash: got ctrl-C
smash: process 111011 was killed
smash> [4] sleep 400& : 111010 8 secs
smash> sleep 400& : 111010 
smash: got ctrl-C
smash: process 111010 was killed
smash> smash> smash: sending SIGKILL signal to 0 jobs:
